x-pack/plugins/security_solution/common/siem_migrations/model/rule_migration.gen.ts

```diff
@@ -88,6 +88,14 @@ export const ElasticRule = z.object({
    * The Elastic prebuilt rule id matched.
    */
   prebuilt_rule_id: NonEmptyString.optional(),
+  /**
+   * The Elastic integration IDs related to the rule.
+   */
+  integration_ids: z.array(z.string()).optional(),
+  /**
+   * The index patterns used by the integrations related to the rule.
+   */
+  index_patterns: z.array(z.string()).optional(),
```

Comment:
Do we needto

Reply:
I was unsure if you somehow wanted the user to have a list of the index patterns used (and modify it), if not I will simply remove it 👍

Reply:
Yes, users will be able to modify index patterns in the query with the editor in the flyout. No need to store it separately 👍